src/libs/ReportUtils.ts

```diff
@@ -4246,6 +4246,21 @@ function doesTransactionThreadHaveViolations(report: OnyxEntry<Report>, transact
     return TransactionUtils.hasViolation(IOUTransactionID, transactionViolations);
 }
 
+/**
+ * Checks if we should display violation - we display violations when the money request has violation and it is not settled
+ */
+function shouldDisplayTransactionThreadViolations(
+    report: OnyxEntry<Report>,
+    transactionViolations: OnyxCollection<TransactionViolation[]>,
+    parentReportAction: OnyxEntry<ReportAction>,
+): boolean {
+    const {IOUReportID} = (parentReportAction?.originalMessage as IOUMessage) ?? {};
+    if (isSettled(IOUReportID)) {
```

Comment:
Later, we added more condition (**approved reports, not only reimbursed/settled**) to hide RBR on the LHN for transaction threads.
https://github.com/Expensify/App/issues/45230